src/apps/relay/mainrelay.h

```diff
@@ -326,6 +326,10 @@ typedef struct _turn_params_ {
   int no_auth_pings;
   int no_dynamic_ip_list;
   int no_dynamic_realms;
+  
+  //// UDP Notifier ////
+  int udp_notifier;
+  char udp_notifier_params[1025];
```

Comment:
1025 is a magic number.

Additionally, does this need to be a flat buffer of characters? can't it be parsed into a struct that has meaningful variable names for whatever the fields are?